doc/bluechi-examples/golang/README.md

```diff
@@ -0,0 +1,35 @@
+# golang examples
```

Comment:
The content of this file should be appended to https://github.com/containers/bluechi/blob/main/doc/docs/api/examples.md as a new section
```
## GoLang

...
```